src/frontend/src/pages/_document.tsx

```
@@ -72,6 +74,13 @@ export default function Document() {
         <Main />
         <NextScript />
         <div id="portal-drag-preview" />
+        {process.env.FRONTEND_INJECT_JS && (
+          <script
+            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)